### PR TITLE
en: add create-task-and-verify.md

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

- add `en/create-task-and-verify.md` 
- remove replicated content in `en/deploy-a-dm-cluster-using-binary.md`
- update TOC.md
- black-white-list --> block-allow-list

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/77
- Other reference link(s):

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
LGTM

File: en/create-task-and-verify.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+## Sample scenario
+
+Suppose that you create a data replication task based on this sample scenario:
+
+- Upstream MySQL instances (MySQL1 and MySQL2) are deployed on two servers, with binlog enabled in both instances.
+- TiDB clusters are deployed on several servers, with the TiDB service exposed on one of the servers.
+- A DM-master of the DM cluster provides service.
+
+The descriptions of each node are as follows.
```
Comment (yikeke) [inline en/create-task-and-verify.md:19]:
```suggestion
The information of each node is as follows.
```

File: en/create-task-and-verify.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+```bash
+./bin/dmctl --encrypt "123456"
+```
+
+```
+fCxfQ9XKCezSzuCD0Wf5dUD+LsKegSg=
+```
+
+Record this encrypted value, which can be used for creating a MySQL data source in the following steps.
```
Comment (yikeke) [inline en/create-task-and-verify.md:48]:
```suggestion
Save this encrypted value, and use it for creating a MySQL data source in the following steps.
```

File: en/create-task-and-verify.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+```bash
+./bin/dmctl --master-addr=192.168.0.4:8261 operate-source create conf/source1.toml
+```
+
+For MySQL2, replace the configuration file in the above command with that of MySQL2.
+
+## Create a data replication task
+
+Suppose that there are several sharded tables on both MySQL1 and MySQL2 instances. In these tables, their structures are identical; the prefix “t” is in their table names; the databases where they are located are named with the same prefix “sharding”; there is no conflict between the primary keys or the unique keys (in each sharded table, the primary keys or the unique keys are different from those of other tables).
```
Comment (yikeke) [inline en/create-task-and-verify.md:85]:
```suggestion
Suppose that there are several sharded tables on both MySQL1 and MySQL2 instances. These tables have identical structure and the same prefix “t” in the table names; the databases where these tables are located are all prefixed with "sharding"; and there is no conflict between the primary keys or the unique keys (in each sharded table, the primary keys or the unique keys are different from those of other tables). 
```

File: en/create-task-and-verify.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+./bin/dmctl --master-addr=192.168.0.4:8261 operate-source create conf/source1.toml
+```
+
+For MySQL2, replace the configuration file in the above command with that of MySQL2.
+
+## Create a data replication task
+
+Suppose that there are several sharded tables on both MySQL1 and MySQL2 instances. In these tables, their structures are identical; the prefix “t” is in their table names; the databases where they are located are named with the same prefix “sharding”; there is no conflict between the primary keys or the unique keys (in each sharded table, the primary keys or the unique keys are different from those of other tables). 
+
+You need to replicate these sharded tables to the `db_target.t_target` table in TiDB.
```
Comment (yikeke) [inline en/create-task-and-verify.md:87]:
```suggestion
Now, suppose that you need to replicate these sharded tables to the `db_target.t_target` table in TiDB. The steps are as follows.
```

File: en/create-task-and-verify.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+    is-sharding: true
+    target-database:
+    host: "192.168.0.3"
+    port: 4000
+    user: "root"
+    password: "" # If the password is not empty, you need to configure the encrypted password using dmctl.
+
+    mysql-instances:
+    - source-id: "mysql-replica-01"
+        block-allow-list:  "instance"  # Using black-white-list if the DM's version <= v2.0.0-beta.2.
```
Comment (yikeke) [inline en/create-task-and-verify.md:106]:
```suggestion
        block-allow-list:  "instance"  # Using block-allow-list if the DM's version <= v2.0.0-beta.2.
```

File: en/create-task-and-verify.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+        route-rules: ["sharding-route-rules-table", "sharding-route-rules-schema"]
+        mydumper-thread: 4
+        loader-thread: 16
+        syncer-thread: 16
+    - source-id: "mysql-replica-02"
+        block-allow-list:  "instance"  # Using black-white-list if the DM's version <= v2.0.0-beta.2.
```
Comment (yikeke) [inline en/create-task-and-verify.md:112]:
```suggestion
        block-allow-list:  "instance"  # Using block-allow-list if the DM's version <= v2.0.0-beta.2.
```

File: en/create-task-and-verify.md [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+        route-rules: ["sharding-route-rules-table", "sharding-route-rules-schema"]
+        mydumper-thread: 4
+        loader-thread: 16
+        syncer-thread: 16
+    block-allow-list:  # Using black-white-list if the DM's version <= v2.0.0-beta.2.
```
Comment (yikeke) [inline en/create-task-and-verify.md:117]:
```suggestion
    block-allow-list:  # Using block-allow-list if the DM's version <= v2.0.0-beta.2.
```

File: en/create-task-and-verify.md [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+            }
+        ]
+    }
+    ```
+
+Now, you have successfully created a task to replicate the sharded tables from the MySQL1 and MySQL2 instances to TiDB.
+
+## Verify whether the cluster works well
+
+You can modify data in the upstream MySQL sharded tables. Then use [sync-diff-inspector](https://docs.pingcap.com/tidb/v4.0/shard-diff) to check whether the upstream and downstream data are consistent. Consistent data means that the replication task works well (this also indicates that the cluster works well).
```
Comment (yikeke) [inline en/create-task-and-verify.md:167]:
```suggestion
You can modify data in the upstream MySQL sharded tables. Then use [sync-diff-inspector](https://docs.pingcap.com/tidb/v4.0/shard-diff) to check whether the upstream and downstream data are consistent. Consistent data means that the replication task works well, which also indicates that the cluster works well.
```

**Comment by toutdesuite** [comment]
@yikeke PTAL again. Thanks

**Pull request merged**